src/actions/WalletActions.tsx

```diff
@@ -227,12 +227,27 @@ const activateWalletTokens = async (
   // See if there is only one wallet option for activation
   if (assetOptions.length === 1 && assetOptions[0].paymentWalletId != null) {
     const { paymentWalletId, tokenId } = assetOptions[0]
-    const activationQuote = await account.activateWallet({
-      activateWalletId: wallet.id,
-      activateTokenIds: tokenIds,
-      paymentWalletId,
-      paymentTokenId: tokenId
-    })
+    let activationQuote: EdgeActivationQuote
+    try {
+      activationQuote = await account.activateWallet({
+        activateWalletId: wallet.id,
+        activateTokenIds: tokenIds,
+        paymentWalletId,
+        paymentTokenId: tokenId
+      })
+    } catch (e: any) {
+      if (e.name === 'InsufficientFundsError' && e.networkFee != null) {
+        // Allow the user to proceed to token activation scene with fee explanation
+        // @ts-expect-error
+        activationQuote = {
+          networkFee: { nativeAmount: e.networkFee, currencyPluginId: wallet.currencyInfo.pluginId, tokenId }
+        }
```

Comment:
I don't really like this solution, since the scene needs a quote. If we are turning the error into a quote, that says we should have probably just returned a quote from the engine. The engine-provided quote will then fail when the user calls  `approve()`, as we expect.